stor/dx.py

```diff
@@ -99,6 +140,17 @@ def wrapper(self):
     realpath = _noop('realpath')
     expanduser = _noop('expanduser')
 
+    def clear_cached_properties(self):
+        """Clears certain cached properties in DXPath objects.
```

Comment:
nit: *all* cached properties, right?

Reply:
Yeah, I mean the four properties in the list are the only ones cached right now